api/buildcraft/api/core/IFakePlayerProvider.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2011-2015, SpaceToad and the BuildCraft Team http://www.mod-buildcraft.com
+ * <p>
+ * The BuildCraft API is distributed under the terms of the MIT License. Please check the contents of the license, which
+ * should be located as "LICENSE.API" in the BuildCraft source code distribution.
+ */
+package buildcraft.api.core;
+
+import com.mojang.authlib.GameProfile;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.WorldServer;
+
+import net.minecraftforge.common.util.FakePlayer;
+
+public interface IFakePlayerProvider {
+    /**
+     * Returns the generic buildcraft fake player. Note that you shouldn't use this anymore, as you should store the
+     * UUID of the real player who created the block or entity that calls this.
+     */
+    @Deprecated
+    FakePlayer getBuildCraftPlayer(WorldServer world);
```

Comment:
Don't add this deprecate method to the API - we'll only remove it later in source code, and I'd rather not keep support for it around in the api.